modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/ChangeTrackingIndicator.js

```diff
@@ -471,7 +479,17 @@ export default function ChangeTrackingIndicator({
 						</ClayLayout.ContentCol>
 					</ClayLayout.ContentRow>
 				}
-				onShowChange={setShowWarning}
+				onShowChange={() => {
+					setShowWarning(false);
```

Comment:
The `onShowChange` is called when the show prop changes. We need to update the `showWarning` according to the state of the prop change otherwise `showWarning` will always be false.

```
onShowChange={(value) => {
	setShowWarning(value);
        ....
}}
```